### ISSUE TITLE
Support readonly mode, disabling upload functions

### ISSUE THREAD

**Issue by aminsbd1** [safrazik/vue-file-agent#35]
Can i disabled/hide button upload? only list file.

**Comment by safrazik** [comment]
If you mean to use just the file previews instead of file inputs, it’s already possible. You can use the provided component `VueFilePreview` for that purpose. Have a look at [src/demo/demo-previews.vue ](https://github.com/safrazik/vue-file-agent/blob/master/src/demo/demo-previews.vue) for example

**Comment by aminsbd1** [comment]
Thankyou!!

**Comment by aminsbd1** [comment]
can't sortable if I use VueFilePreview?

**Comment by safrazik** [comment]
`VueFilePreview` is an internal component meant for each file's preview handling. I haven't documented it yet until I find better use cases.

Looking at your use case, I think it would be better to add a `readonly` prop to the `VueFileAgent` component. For the time being, you can achieve your intended behavior with CSS. (Hopefully next version will have this in built)

```vue
<template>
  <VueFileAgent 
    class="is-readonly"
  </VueFileAgent>
</template>
<style>
  .is-readonly .vue-file-agent .file-preview-new {
    display: none;
  }
  .is-readonly .vue-file-agent .file-input {
    display: none;
  }
</style>
```

**Comment by safrazik** [comment]
Available since `v 1.5.3-beta.2`
Added in ca53bdeef6921e9a3728c8d66d9ecb0b06c40814
